projects/azure/README.md

```diff
@@ -288,10 +288,8 @@ config to onboard building the image in CI
 * If you intend for the image to be mirrored from the CI registry to the public quay registry then you need to update the
 [image mirror config](https://github.com/openshift/release/blob/master/projects/azure/image-mirror/image-mirror.yaml) 
 in the release repo
-	* Add a new entry to the `items[0].data.openshift-azure_v3_11_quay` key of that file which looks as follows:
-	```yaml
-	registry.svc.ci.openshift.org/azure/azure-plugins:<image_name> quay.io/openshift-on-azure/<image_name>:v3.11 quay.io/openshift-on-azure/<image_name>:latest
```

Comment:
My comment was about simply removing `quay.io/openshift-on-azure/<image_name>:v3.11`, nothing less, nothing more. :)

Reply:
I would like to document the fact that images can and should have the openshift version associated. I believe the latest changes achieve this.

Reply:
Note that we use the openshift version as a weak convention at the moment really, we haven't agreed that versioning our images based on openshift is what we really want to do.